packages/headless/src/controllers/commerce/core/facets/generator/headless-commerce-facet-generator.test.ts

```diff
@@ -8,8 +8,10 @@ import {
   buildMockCommerceDateFacetResponse,
```

Comment:
I've migrated / updated these tests here: https://github.com/coveo/ui-kit/pull/3616/files#diff-443d10e9e85bf523fb7331c2ef00fe2a4b0b92ba44ad742bb427d8693a39c5fb

So if you want you can revert the changes you made in this file.